src/assets/metadata/aws.json

```diff
@@ -1,311 +1,3661 @@
+          "url": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cloudwatch_metric_alarm#alarm_actions",
+          "rules": {
+            "max": 5,
+            "min": 1
+          }
+        },
+        {
+          "name": "insufficient_data_actions",
+          "displayName": "Insufficient data actions",
+          "description": "The list of actions to execute when this alarm transitions into an INSUFFICIENT_DATA state from any other state.",
+          "type": "Number",
+          "url": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cloudwatch_metric_alarm#insufficient_data_actions",
+          "rules": {
+            "max": 5,
+            "min": 1
+          }
+        },
+        {
+          "name": "ok_actions",
+          "displayName": "OK actions",
+          "description": "The list of actions to execute when this alarm transitions into an OK state from any other state.",
+          "type": "Number",
+          "url": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cloudwatch_metric_alarm#ok_actions",
+          "rules": {
+            "max": 5,
+            "min": 1
+          }
+        },
+        {
+          "name": "unit",
+          "displayName": "Unit",
+          "description": "The unit for the alarm's associated metric.",
+          "type": "String",
+          "url": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cloudwatch_metric_alarm#unit",
+          "rules": {
+            "values": [
+              "Seconds",
+              "Microseconds",
+              "Milliseconds",
+              "Bytes",
+              "Kilobytes",
+              "Megabytes",
+              "Gigabytes",
+              "Terabytes",
+              "Bits",
+              "Kilobits",
+              "Megabits",
+              "Gigabits",
+              "Terabits",
+              "Percent",
+              "Count",
+              "Bytes/Second",
+              "Kilobytes/Second",
+              "Megabytes/Second",
+              "Gigabytes/Second",
+              "Terabytes/Second",
+              "Bits/Second",
+              "Kilobits/Second",
+              "Megabits/Second",
+              "Gigabits/Second",
+              "Terabits/Second",
+              "Count/Second",
+              "None"
+            ]
+          }
+        },
+        {
+          "name": "extended_statistic",
+          "displayName": "Extended statistic",
+          "description": "The percentile statistic for the metric associated with the alarm. Specify a value between p0.0 and p100.",
+          "type": "String",
+          "url": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cloudwatch_metric_alarm#extended_statistic"
+        },
+        {
+          "name": "treat_missing_data",
+          "displayName": "Treat missing data",
+          "description": "Sets how this alarm is to handle missing data points.",
```

Comment:
suggestion : `Defines how this alarm should handle missing data points.`

Reply:
description from official doc.